examples/webpack-jsx/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'cheap-module-source-map',
+  entry: './src/index',
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+    publicPath: '/dist/'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader'
+      },
+      {
+        test: /\.css$/,
+        loader: 'style-loader!raw-loader'
+      },
+      {
+        test: /\.png$/,
+        include: path.join(__dirname, 'assets'),
+        loader: 'url-loader?mimetype=image/png'
+      },
+    ]
+  },
+  resolve: {
+    alias: {
+      react: path.resolve('./node_modules/react')
```

Comment:
I've had to add this is as it was pulling in 2 versions of react, I think maybe the instance of React located at the root of the Spectacle directory as well as the version defined by `webpack-jsx/package.json`. I'm not entirely sure why this is happening so adding this in resolves all imports of react to the version defined in `webpack-jsx`'s `node_modules` folder.